src/components/Categories.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Categories = () => <p>Under construction</p>;
+
+export default Categories;
```

Comment:
- [ ] This component should be  done as `Route or Link` see the screenshot 👇.

![kklkl](https://user-images.githubusercontent.com/61562626/132661779-c695d6db-72bf-4151-ab7c-b3ce1f5b6730.jpeg)